eg01-eg23.json

```diff
@@ -0,0 +1,656 @@
+      "mod_Sig_a": "0.02220", 
+      "mod_Sig_f": "0.0", 
+      "mod_Sig_tr": "3.460", 
+      "mod_thickness": "100", 
+      "nonleakage": "0.57", 
+      "out_commod": "FR Spent Fuel", 
+      "reactor_life": "960", 
+      "target_burnup": "200", 
+      "tolerence": "0.0010", 
+      "trans_created": {
+       "val": [
+        "922350", 
+        "942380", 
+        "942390", 
+        "942400", 
+        "942410", 
+        "942420"
+       ]
+      }, 
+      "trans_fission": {
+       "val": [
+        "922350", 
+        "942380", 
+        "942390", 
+        "942400", 
+        "942410", 
+        "942420"
+       ]
+      }
+     }
+    }, 
+    "name": "FR"
+   }, 
+   {
+    "config": {
+     "ReprocessFacility": {
+      "commod_out": {
+       "val": [
+        "FR Reprocessed", 
+        "WASTE"
+       ]
+      }, 
+      "in_commod": {
+       "val": "FR Spent Fuel"
+      }, 
+      "input_capacity": "2000", 
+      "max_inv_size": "1.000000000000000E+299", 
+      "output_capacity": "2000", 
+      "repro_input_path": "hist/FR_reprocess.txt"
+     }
+    }, 
+    "name": "FR Reprocess"
+   }, 
+   {
+    "config": {
+     "Sink": {
+      "capacity": "100000", 
+      "in_commods": {
+       "val": "WASTE"
+      }, 
+      "max_inv_size": "1.000000000000000E+299"
+     }
+    }, 
+    "name": "SINK"
+   }
+  ], 
+  "recipe": [
+   {
+    "basis": "mass", 
+    "name": "Uranium 238", 
+    "nuclide": {
+     "comp": "100.0000002", 
+     "id": "922380"
+    }
+   }, 
+   {
+    "basis": "mass", 
+    "name": "U235", 
+    "nuclide": {
+     "comp": "100.0000002", 
+     "id": "922350"
+    }
+   }, 
+   {
+    "basis": "mass", 
+    "name": "DU", 
+    "nuclide": [
+     {
+      "comp": "0.2500002", 
+      "id": "922350"
+     }, 
+     {
+      "comp": "99.7500002", 
+      "id": "922380"
+     }
+    ]
+   }
+  ], 
+  "region": {
+   "config": {
+    "NullRegion": null
+   }, 
+   "institution": [
+    {
+     "config": {
+      "NullInst": null
+     }, 
+     "initialfacilitylist": {
+      "entry": [
+       {
+        "number": 1, 
+        "prototype": "MineU235"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "U238"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "DU"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "DU2"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "LWR Fuel FAb"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "LWR Seperation"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "FR Reprocess"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "SINK"
+       }, 
+       {
+        "number": 1, 
+        "prototype": "FR Fuel Fab"
+       }
+      ]
+     }, 
+     "name": "utility"
+    }, 
+    {
+     "config": {
+      "DeployInst": {
+       "buildorder": [
+        "<prototype>LWR</prototype><number>1</number><date>6</date>", 
```

Comment:
Interested in why these dates are in year rather than month. Is that handled elsewhere through the json sim def stuff?